Mute default :focus form highlight (#36103)
mdn/content: files/en-us/learn/forms/your_first_form/index.md

input:focus,

> textarea:focus {
>   /* Mute default highlight */
>   outline: none;
>   /* Additional highlight for focused elements */
>   border-color: #000;
> }

Reviewer: This is what was agreed in thread.

```suggestion
textarea:focus {
  /* To give a little highlight on active elements */
  outline-color: #000;
}
```

input:focus,
textarea:focus {

>   /* Set the outline width and style /

Reviewer: ```suggestion
  /* Set the outline width and style */
```
